include/zio/zio_dev.h

```diff
@@ -80,14 +79,21 @@ struct zio_buf;
  * @brief Pre-defined channel types with the ability to extend by a driver
  * implementation.
  *
- * Extended values should always be based on ZIO_LAST_CHAN_TYPE with an
+ * Extended values should always be based on ZIO_CHAN_TYPES count with an
  * additive and defined in a publicly available driver header.
+ *
+ * Example: #define MY_CHAN_TYPE (ZIO_CHAN_TYPES+1)
```

Comment:
``#name`` in a doxygen comment is used to reference a variable named "name", hence the error that's showing up saying:
```
warning: explicit link request to 'define' could not be resolved
```
Workaround is to escape the ``#`` with ``\#``

```suggestion
 * Example: \#define MY_CHAN_TYPE (ZIO_CHAN_TYPES+1)
```